pkg/apis/hive/v1alpha1/validating-webhooks/syncset_validating_admission_hook.go

```diff
@@ -151,15 +152,21 @@ func (a *SyncSetValidatingAdmissionHook) validateCreate(admissionSpec *admission
 	// Add the new data to the contextLogger
 	contextLogger.Data["object.Name"] = newObject.Name
 
-	if invalid, ok := checkValidPatchTypes(newObject.Spec.Patches); !ok {
-		message := fmt.Sprintf("Failed validation: Invalid patch type detected: %s. Valid patch types are: json, merge, and strategic", invalid)
-		contextLogger.Infof(message)
+	if allErrs := validateResources(newObject.Spec.Resources, field.NewPath("spec").Child("resources")); len(allErrs) > 0 {
```

Comment:
All the errors from `validateResources`, `validatePatches`, and `validateSecrets` should be aggregated together instead of returning when the are errors from one of them.